template.go

```diff
@@ -299,7 +300,12 @@ func writeDebianRules(dir string, pkgType packageType) error {
 	fmt.Fprintf(f, "#!/usr/bin/make -f\n")
 	fmt.Fprintf(f, "\n")
 	fmt.Fprintf(f, "%%:\n")
-	fmt.Fprintf(f, "\tdh $@ --builddirectory=_build --buildsystem=golang\n")
+	fmt.Fprintf(f, "\tdh $@ --builddirectory=debian/build --buildsystem=golang\n")
+	// Note: The above `--builddirectory=debian/build` will eventually be obsolete
+	// in 2028+ then the dh-golang version 1.63+ that has
+	// https://salsa.debian.org/go-team/packages/dh-golang/-/commit/bc16dff5381b668a71fa99c381baba202c34c789
+	// is in use everywhere
+
```

Comment:
Considering `dh-golang` 1.63 is already in testing, I think we could remove this completely. After all, these tools are meant to be used from sid..

Reply:
The omission of the --builddirectory option is dependent on the debhelper compat level, see the dh-golang(1) man page for the rationale, and debhelper compat 14 is not yet stable.